src/search.c

```diff
@@ -53,6 +55,36 @@ find_matches(struct view *view)
 	return true;
 }
 
+static bool
+find_merges(struct view *view)
+{
+	size_t lineno;
+
+	for (lineno = 0; lineno < view->lines; lineno++) {
+		struct line *line = &view->line[lineno];
+		struct commit *commit = line->data;
+		struct graph_canvas *canvas = &commit->graph;
+		struct view_column_data column_data;
+
+		if (!view->ops->get_column_data(view, line, &column_data))
+			continue;
+
+		if (!column_data.graph->is_merge(canvas))
+			continue;
+
+		if (!realloc_unsigned_ints(&view->matched_line, view->matched_lines, 1))
+			return false;
+
+		view->line[lineno].search_result = true;
+		view->matched_line[view->matched_lines++] = lineno;
```

Comment:
Any particular reason to reuse these member that are usually for search highlighting?